<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="name_desc_tags_placeholder" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="device_widget_placeholder" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="comparisons_table">
     <column>
      <property name="text">
       <string>Comparisons</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_comparison_button">
     <property name="text">
      <string>+ Add Comparison</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
